tools/clang/tools/dxil2spv/lib/dxil2spv.h

```diff
@@ -97,6 +97,9 @@ class Translator {
   // SPIR-V Tools wrapper functions.
   bool spirvToolsValidate(std::vector<uint32_t> *mod, std::string *messages);
 
+  // Output SPIR-V module.
+  void outputSpirvModule(std::vector<uint32_t> spirvModule);
```

Comment:
Could we pass the spir-v code by reference so that we don't copy the data. I think you can use `ArrayRef<uint32_t>` for that.

Reply:
Done